ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gojek/mlp-ui",
-  "version": "1.4.4",
+  "version": "1.4.5",
```

Comment:
I think it should be `1.4.6` right? Since we currently already have `1.4.5` tagged.